### ISSUE TITLE
Sorry,is there something wrong with the code?

### ISSUE THREAD

**Issue by kkizjw** [ViktorAxelsen/CLE-TFE#1]
when i run the code ,they report"C:\cb\pytorch_1000000000000\work\aten\src\ATen\native\cuda\Loss.cu:250: block: [0,0,0], thread: [6,0,0] Assertion `t >= 0 && t < n_classes` failed.
C:\cb\pytorch_1000000000000\work\aten\src\ATen\native\cuda\Loss.cu:250: block: [0,0,0], thread: [9,0,0] Assertion `t >= 0 && t < n_classes` failed." I think there is something wrong with the loss function.
